library/src/scripts/headers/navigationVariables.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+import { t } from "@library/utility/appUtils";
+
+export const navigationVariables = () => {
+    return {
+        en: {
+            data: [
+                {
+                    to: "/kb",
+                    children: t("Help Menu", "Learning Center"),
+                    permission: "kb.view",
+                },
+                {
+                    to: "/discussions",
+                    children: t("Discussions"),
+                },
+                {
+                    to: "/categories",
+                    children: t("Innovation Hub"),
```

Comment:
What is `Innovation Hub`? That is first time I hear such term.

Reply:
It's the client wanting to call their categories page that.

Reply:
But we are in vanilla core repo now.

Reply:
Woah, this is definitely in the wrong repo. That should be in the custom theme, not in core.